HinojosaFernando/src/BookType.java

```diff
@@ -0,0 +1,3 @@
+public enum BookType {
+    scientific, biography, recreational, poetic, tragedy, fantasy, adventure, memories, selfhelp;
```

Comment:
Sorry, I forgot to mention but Enum items are usually in UPPER_CASE. That way is easier to identify them and distinguish them from class attributes.